### DISCUSSION TITLE
Add manual updates instead forced updates like GitHub Atom

### DISCUSSION THREAD

**Discussion by Neustradamus** [community/community#24087] [Apps, API and Webhooks]
<p>Currently in GitHub Desktop, the auto-update is enabled by default and can not be disabled.</p>
<p>Can you add an option like GitHub Atom to update manually and not forced?</p>
<ul>
<li><a href="https://i.ibb.co/3mmC0Gz/github-atom-manual-updates.png" rel="noopener nofollow ugc">https://i.ibb.co/3mmC0Gz/github-atom-manual-updates.png</a></li>
</ul>

**Comment by steveward** [comment, marked as answer]
<p>Thanks for the feedback. We are tracking this in <a href="https://github.com/desktop/desktop/issues/3410" class="inline-onebox">Opt-out of auto-updating new app versions · Issue #3410 · desktop/desktop · GitHub</a>.</p>

**Comment by Neustradamus** [comment]
<p><a class="mention" href="/steveward">@steveward</a>: Thanks for your reply!</p>
<p>The issue exist since 23 Nov 2017, maybe time to look no?</p>

**Comment by kingthorin** [comment]
<aside class="quote no-group" data-username="Neustradamus" data-post="3" data-topic="156750">
<div class="title">
<div class="quote-controls"></div>
 Neustradamus:</div>
<blockquote>
<p>The issue exist since 23 Nov 2017, maybe time to look no?</p>
</blockquote>
</aside>
<p>In a perfect world everyone’s gutt feel big deal or simple items would be tackled right away… Unfortunately the world isn’t perfect, issues aren’t small or simple just because requestor wish it so. The software is open, you can submit a PR implementing the feature if you like. If you don’t like or aren’t able then you’re kind getting what you paid for… :wink:</p>
<p>Great it’s been open for 3 years, there’s also a bunch of tickets that have been open for four years. If age is actually relevant and they tackle it as first in first out, this one is likely still a ways away. On top of that you are kind of assuming this is relevant to a major portion of their user base, yet only 8 to 10 people have bothered to comment on the issue, maybe as many as like -40 if we also count thumbs ups and hearts. Out of 100s of 1000s of users that’s kinda diddly squat.</p>
<p>I’m not saying it shouldn’t be addressed, I’m just trying to play devil’s advocate a bit and get you to see/consider other perspectives, priorities, workloads, interests, etc.</p>

**Comment by steveward** [comment]
<p><a class="mention" href="/Neustradamus">@Neustradamus</a> we have a lot of open issues – both bugs and feature requests. We are a small team and have to prioritize what we work on, but thanks letting us know you’d like to see this implemented.</p>
